dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -91,21 +98,38 @@ const supportingText = css`
 	color: ${neutral[46]};
 `;
 
-interface Props {
+export const CalloutShare = ({
+	format,
+}: {
 	format: ArticleFormat;
-}
-
-export const CalloutShare = ({ format }: Props) => {
+	title: string;
+}) => {
 	const [isCopied, setIsCopied] = useState(false);
 
 	const onShare = async () => {
 		const url = window.location.href;
+		if ('share' in navigator) {
```

Comment:
@sophie-macmillan here we check if the navigator is able to "share" (for example, Safari on IOS is able to share) and if so we invoke the share sheet. If not, we just copy the link to clipboard (see line 126)